packages/fxa-settings/src/pages/InlineTotpSetup/container.tsx

```diff
@@ -146,6 +147,7 @@ export const InlineTotpSetupContainer = ({
           ...Object.assign({}, signinState),
           ...(totp ? { totp } : {}),
         };
+        GleanMetrics.accountPref.twoStepAuthQrCodeSuccess();
```

Comment:
During MozWeek, Valerie and I chatted with Ross, and he's okay with the two 2fa setups (settings and inline) use the same events.